go-console/.conf/shareWSLPorts.ps1

```diff
@@ -0,0 +1,46 @@
+
+# this only makes sense for WSL
```

Comment:
This file can be removed, this is automatically copied from scripts to the project folder during the creation.